src/pages/JobTracking.jsx

```diff
@@ -1,19 +1,26 @@
 import React from "react";
 import ApplicationForm from "../components/ApplicationForm";
+import { NavLink } from "react-router";
 
 const JobTracking = () => {
   return (
     <div className=" bg-softPink flex flex-col items-left justify-start p-5 gap-5">
       <div className="flex flex-row">
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8 w-1/2 text-left hover:border-2 hover:border-solid hover:border-blue-500">
-          Job Tracking
-        </div>
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
-          Home
-        </div>
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
-          Resources
-        </div>
+        <NavLink to={"/jobtracking"} className="w-1/2 gap-5">
+          <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8  text-left hover:border-2 hover:border-solid hover:border-blue-500">
+            Job Tracking
+          </div>
+        </NavLink>
```

Comment:
```suggestion
          <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8  text-left hover:border-2 hover:border-solid hover:border-blue-500">
            Job Tracking
          </div>
```

Same with this -- it is just a label, not a nav button